sdk/communication/Azure.Communication.CallAutomation/src/Models/TransferToParticipantOptions.cs

```diff
@@ -13,16 +14,47 @@ public class TransferToParticipantOptions
         /// <summary>
         /// Creates a new TransferToParticipantOptions object.
         /// </summary>
-        /// <param name="callInvite"></param>
-        public TransferToParticipantOptions(CallInvite callInvite)
+        /// <param name="targetPhoneNumberIdentity"></param>
+        /// <param name="sipHeaders"></param>
+        public TransferToParticipantOptions(PhoneNumberIdentifier targetPhoneNumberIdentity, IDictionary<string, string> sipHeaders = null)
         {
-            CallInvite = callInvite;
+            Target = targetPhoneNumberIdentity;
+            SipHeaders = sipHeaders == null ? new Dictionary<string, string>() : sipHeaders;
         }
 
         /// <summary>
-        /// Call invitee information.
+        /// Creates a new TransferToParticipantOptions object.
+        /// </summary>
+        /// <param name="targetIdentity"></param>
+        /// <param name="voipHeaders"></param>
+        public TransferToParticipantOptions(CommunicationUserIdentifier targetIdentity, IDictionary<string, string> voipHeaders = null)
```

Comment:
You dont need optional input in signature for simple method - simple method to only accept required field only. You can remove voip headers here.

if dev needs to use voip headers, they need to use options